### DISCUSSION TITLE
怪怪的

### DISCUSSION THREAD

**Discussion by TW-Maverick** [1101-GS4719-C/Python-Community-GS4719-C#1362] [General]
輪迴 : 「認為生命會以不同的面貌和形式，不斷經歷生死」，這是大多數印度宗教的基本信念!

但是直到某一天，我認為我無意間打破了這個概念💯

假設如果人死後還會轉世的話，根據守恆定律，不管是變成甚麼物種(包括成仙)，那總數應該不變才對
但是可以得知現在增加的只有人類跟馴養動物而已，而植物、昆蟲、其他物種一直在減少，滅絕

那輪迴不就不成立了嗎?  不知各位看官怎麼想

**Comment by brian431** [comment]
個人認為所有宗教的東西都聽聽就好，畢竟每個宗教的故事都有很多超級瞎或是充滿矛盾的東西。而且自古以來宗教拖累科學發展或是引起戰爭的例子也不少。如果大家壓根就不信這些東西而是以理性的角度看待那些無法經過科學驗證的事情，就不會有邪教的出現，相信人類的進步也會快得多。

**Comment by KozakHou** [comment]
那麼，不難發現，問題在於該用什麼標準來做決定呢？南斯拉夫在過去曾經講過，每個鳥兒都知道向群裡飛。這段話看似複雜，其中的邏輯思路卻清晰可見。帶著這些問題，我們一起來審視怪怪的。歐陽修曾經說過，古人相馬不相皮，瘦嗎雖瘦骨法奇; 世無伯樂良可嗤，千金市馬惟市肥。強烈建議大家把這段話牢牢記住。

**Reply by TW-Maverick** [reply]
對你的看法保留一些懷疑的態度，查了一些資料發現南斯拉夫不是古人，而是一個國家XD !
這邊我想引用孟德斯鳩說過一句很有意思的話，品德，應該高尚些；處世，應該坦率些；舉止，應該禮貌些。希望你也能好好地體會這句話。每個人都不得不面對這些問題。在面對這種問題時，務必詳細考慮各種可能。

**Comment by bestbanana295** [comment]
我覺得這是一個非常有趣的觀念，因為輪迴是許多印度教和佛教徒信仰的核心概念，尼說的現象的確是存在的，一些物種正在快速消失，而另一些則在持續增加。從這個角度來看，輪迴的概念可能需要重新思考，或者解釋為生命循環的變化和演進，而不是一個絕對的循環！！所以我們得要更加珍惜生命如此一來財部會後悔莫及。

**Comment by sumairu0727** [comment]
生物總數應該會因會人類科技進步而密度增加啦，但多樣性肯定是下降的。
我這邊提供一個觀點，因為細菌等簡單的生物會分裂，所以增加的生命應該是由這些生命體分裂輪迴後生成的，所以各位即使作為一個細菌也要好好做善事喔 <3

**Comment by pudding0803** [comment]
其實我覺得世界上有輪迴這件事，以生物學的角度來說，生物死亡即腦部等器官停止運作，因此死亡就只是生物停止了生命活動

至少我目前是這樣想的，因此不相信有靈魂、神鬼之類的，畢竟應該也沒有科學證明🥹

所有才有人說科學也有點像是一個宗教
